recipes/flann/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set commit = "40d63fc7db5d1f6dd67364f24e4579aedb57e7b6" %}
+
+{% set tag = environ.get("GIT_DESCRIBE_TAG", "") %}
+{% set number = environ.get("GIT_DESCRIBE_NUMBER", "0") %}
+{% set hash = environ.get("GIT_DESCRIBE_HASH", "0") %}
+{% set tag_ext = ("+" + number + "." + hash) if number != "0" else "" %}
+{% set version = tag + tag_ext %}
+
+
+package:
+    name: flann
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/mariusmuja/flann.git
+    git_tag: {{ commit }}
+
+build:
+    number: 0
+    skip: True    # [win]
+
+requirements:
+    build:
+        - gcc     # [osx]
+        - hdf5
+        - cmake
+    run:
+        - libgcc  # [osx]
+        - hdf5
+
```

Comment:
So, normally we add a `test` section here. In practice, we go for something pretty simple like making sure all the libraries are there. Exercising help or version info from any binaries installed.